dagger-functions/src/test/java/io/odpf/dagger/functions/udfs/python/file/type/PythonFileTypeTest.java

```diff
@@ -0,0 +1,23 @@
+package io.odpf.dagger.functions.udfs.python.file.type;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.File;
+import java.util.List;
+import java.util.Objects;
+
+public class PythonFileTypeTest {
+
+    @Test
+    public void shouldGetFileNames() {
+        ClassLoader classLoader = getClass().getClassLoader();
+        File file = new File(Objects.requireNonNull(classLoader.getResource("test_udf.py")).getFile());
+        String pathFile = file.getAbsolutePath();
+
+        PythonFileType pythonFileType = new PythonFileType(pathFile);
+        List<String> fileNames = pythonFileType.getFileNames();
+
+        Assert.assertEquals("[test_udf.py]", fileNames.toString());
+    }
```

Comment:
can we add failure scenario tests, like what happens if pathFile is empty, null?

Reply:
btw, i will change all the variable name of pathFile to pythonFile. if its null, it will throw PythonFilesNullException. this is handled in PythonUdfManagerClass